types/ramda/index.d.ts

```diff
@@ -1869,8 +1870,10 @@ export function once<F extends (...a: readonly any[]) => any>(fn: F): F;
  * A function that returns the first truthy of two arguments otherwise the last argument. Note that this is
  * NOT short-circuited, meaning that if expressions are passed they are both evaluated.
  */
-export function or<T, U>(a: T, b: U): T | U;
-export function or<T>(a: T): <U>(b: U) => T | U;
+export function or<U>(a: Falsy, b: U): U;
+export function or(a: Falsy): <U>(b: U) => U;
+export function or<T, U>(a: T | Falsy, b: U): T | U;
+export function or<T>(a: T | Falsy): <U>(b: U) => T | U;
```

Comment:
What the reason/advantage for having the first two overloads? Do they add anything that isn't handled by the later two?

Reply:
On second thought... yeah they aren't too useful.
I just added them because the tests have cases in which the first parameter is `0` (i.e. just `R.Falsy`), but I think that should never happen in any sensible code anyway

Reply:
That said, I need to add tests where `T` is a union... oops